hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/compact/CompactContext.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.serialization.impl.compact;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/**
+ * Allows access to schemas to services outside of the {@link CompactStreamSerializer}
+ */
+public class CompactContext {
+
+    private final CompactStreamSerializer compactStreamSerializer;
+
+
+    public CompactContext(CompactStreamSerializer compactStreamSerializer) {
+        this.compactStreamSerializer = compactStreamSerializer;
+    }
+
+    @Nullable
+    public Schema getSchema(@Nonnull String typeName) {
```

Comment:
It does not seem to be used 🤔

Reply:
Yes, after refactoring the code it seems it is not needed anymore.